### DISCUSSION TITLE
Variant viewer should link to UCSC Browser

### DISCUSSION THREAD

**Discussion by diekhans** [KarchinLab/open-cravat#245] [Usability & Feature Requests]
It is important to be able to quickly explore data in genomic context.  Being able to quickly jump to the variant view in the UCSC browser would be a highly valuable feature.

**Comment by kmoad** [comment]
Working on an annotator that provides these links.

**Comment by kmoad** [comment]
We've added an annotator `ucscgenomebrowser` which provides a link to the variant location in the genome browser.
